docs/manuals/core/config/data-node-config.md

```diff
@@ -772,4 +772,30 @@ The scope is `SCENARIO` (default value), and default data is provided.
     `Config.configure_in_memory_data_node()^` or the method `Config.configure_data_node()^`
     with parameter `storage_type="in_memory"`.
 
+# Default data node configuration
+
+By default, if you don't provide any information when configure a datanode (except for the mandatory _**id**_), the `Config.configure_data_node()^` method will return a _pickle_ data node configuration with `SCENARIO` scope.
+
+To override the default data node configuration, one can use the `Config.configure_default_data_node()^` method.
+Then, a new data node configuration will:
+
+- have the same properties as the default data node if the _**storage_type**_ is the same.
+- ignore default data node configuration if the _**storage_type**_ is different.
+
+```python linenums="1"
+{%
+include-markdown "./code_example/data_node_cfg/data-node-config_default.py"
+comments=false
+%}
+```
+
+We override the default data node by a SQL table data node configuration in the previous code example, providing all necessary properties for a SQL table data node in lines 3-14.
+
+Then we configure 5 data nodes:
+
+- Line 16 configures a SQL Table data node `products_data_cfg`. By providing only the _**id**_, `products_data_cfg` has the exact same properties as the default data node configuration as above, which reads and writes to "products" table.
+- Line 17 configures a SQL Table data node `users_data_cfg`. By also providing `table_name="users"`, this data node reads and writes to "users" table.
```

Comment:
"configures a SQL" -> "configures the SQL"
"writes to "users" table" -> "writes to the "users" table"